planning/standards-status.md

```diff
@@ -0,0 +1,83 @@
+| [NNTP](https://datatracker.ietf.org/doc/html/rfc977)               |  &#10004;      |   &#10004;     |                            |         |
+
+## RSS Feeder
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [RSS](https://www.rssboard.org/rss-specification)                  |  &#10004;      |      N/A       |                            | RSS 0.9 vs 1.0 vs 2.0 are rather different. Which do we support?  |
+| [Atom](https://www.rfc-editor.org/rfc/rfc4287)                     |  &#10004;      |      N/A       |                            |         |
+
+## Security and Authentication
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [MIME](https://www.rfc-editor.org/rfc/rfc2045)                     |  &#10004;      |   &#10004;     |                            | MIME is actually over 3 RFCs: [RFC2045](https://www.rfc-editor.org/rfc/rfc2045), [RFC2046](https://www.rfc-editor.org/rfc/rfc2046), and [RFC2047](https://www.rfc-editor.org/rfc/rfc2047). All 3 are supported for both Desktop and Android platforms. |
+| [List Management Headers](https://www.rfc-editor.org/rfc/rfc2369)  |                |                |                            | See [Bug 29041](https://bugzilla.mozilla.org/show_bug.cgi?id=29041)  |
+| OpenPGP                                                            |  &#10004;      |   &#10004;     | [KaiE]() actively involved |         |
+| S/MIME version?                                                    |  &#10004;      |                |                            |         |
+| [oAuth 2.0](https://datatracker.ietf.org/doc/html/rfc6749)         |  &#10004;      |   &#10004;     |                            | See https://oauth.net/2/  |
+| [LDAP](https://datatracker.ietf.org/doc/html/rfc4511)              |  &#10004;      |      N/A       |                            |         |
+
+## Address Book
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [vCard](https://datatracker.ietf.org/doc/html/rfc6350)             |  &#10004;      |                |                            |         |
+| [CardDAV](https://datatracker.ietf.org/doc/html/rfc4511)           |  &#10004;      |      N/A       |                            |         |
```

Comment:
This is probably missing LDAP

Reply:
I moved LDAP to the authentication and security section but I am happy to move it back to the Address Book section if you think it makes more sense here.

Reply:
We do use LDAP as actual address book "directories" and not for auth.

Reply:
ok gotcha. I've moved LDAP back to the address book section.